packages/steps/src/cacheUtils.ts

```diff
@@ -0,0 +1,31 @@
+import { bunyan } from '@expo/logger';
+
+export interface Cache {
```

Comment:
Do we need to have two `Cache`s, one here and another one in `eas-build-job`?

Reply:
It is part of the transition process: in order to avoid circular dependencies `Cache` should be defined in `steps`, but to keep compatibility I kept it in both places. It will be consolidated after changes to turtle

Reply:
Is this still true after recent changes to Turtle? Why would we get a circular dependency if we used `Cache` from `eas-build-job`?